thoth/license_solver/classifiers.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python3
+# license-solver
+# Copyright(C) 2021 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and / or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Class download classifiers and extract data from them."""
+
+import os
+import re
+import attr
+import requests
+import logging
+from typing import List, Any
+
+_LOGGER = logging.getLogger(__name__)
+
+
+@attr.s
+class Classifiers:
+    """Class detect all classifiers from downloaded data."""
+
+    received_text = attr.ib(init=False, type=str)
+    classifiers: List[str] = list()
+    classifiers_list: List[Any] = list()
+
+    def __attrs_post_init__(self) -> None:
+        """INIT method."""
+        self._download_classifiers()
```

Comment:
Could loading of classifiers be done from a file instead of accessing them from the internet? Downloading them each time might slow down multiple runs.